scaler/index.js

```diff
@@ -49,9 +49,9 @@ async function scaleSpannerInstance(spanner, suggestedNodes) {
       'INFO');
 
   const metadata = {
-    // displayName : 'instance' + Math.floor(Math.random() * 100) // For
-    // testing. See next line for actual scaling
-    nodeCount: suggestedNodes
+    // For testing. See next line for actual scaling
+     displayName : 'instance' + Math.floor(Math.random() * 100)
+    //nodeCount: suggestedNodes
```

Comment:
This looks like a testing change.  Did you intend to commit this?

Reply:
good catch !
Fixed and added unit test to avoid it from happening again.